### ISSUE TITLE
Invalid asset ID

### ISSUE THREAD

**Issue by prabint** [nasa/NASA-APIs#23]
Invalid asset ID: https://images-api.nasa.gov/asset/The N.E.X.T. Thing for Space Travel
Returns 403 on https://images.nasa.gov/.

**Comment by JustinGOSSES** [comment]
Can you be more specific? Why do you think that asset ID should be valid? What is the behavior you'd expect and why?

**Comment by prabint** [comment]
I am fetching list of videos and one of the result's response is this:

**Request**: 
```https://images-api.nasa.gov/search?q=Thing for Space Travel&media_type=video```

**Response**: 
```
{
    "collection": {
        "metadata": {
            "total_hits": 1
        },
        "href": "https://images-api.nasa.gov/search?q=Thing%20for%20Space%20Travel&media_type=video",
        "version": "1.0",
        "items": [
            {
                "links": [
                    {
                        "rel": "captions",
                        "href": "https://images-assets.nasa.gov/video/The N.E.X.T. Thing for Space Travel/The N.E.X.T. Thing for Space Travel.srt"
                    }
                ],
                "href": "https://images-assets.nasa.gov/video/The N.E.X.T. Thing for Space Travel/collection.json",
                "data": [
                    {
                        "keywords": [
                            "Space Exploration",
                            "Ion Propulsion"
                        ],
                        "description": "The NASA Evolutionary Xenon Thruster or NEXT is an advanced Ion propulsion system developed at Glenn Research Center. Its unmatched fuel efficiency could give a real boost to future deep space exploration missions -- extending the reach of NASA science missions and yielding a higher return on scientific research.",
                        "nasa_id": "The N.E.X.T. Thing for Space Travel",
                        "media_type": "video",
                        "date_created": "2013-07-26T00:00:00Z",
                        "center": "HQ",
                        "title": "The N.E.X.T. Thing for Space Travel"
                    }
                ]
            }
        ]
    }
}
```
Then I am trying to get assets for id `https://images-api.nasa.gov/asset/The N.E.X.T. Thing for Space Travel`
At this point, it returns 403 access denied:
```
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>2E3320D046C3919A</RequestId>
<HostId>
tGgFsjVw+nO+v8yX3euDb5R9j/JPCB1+8tzuo09M8yzNdNYV2mwRCCaNYJLmKwjMDnRc7SXp9zU=
</HostId>
</Error>
```
I tested it out using chrome debugger network tab on Nasa's website: 
`https://images.nasa.gov/search-results?q=Thing%20for%20Space%20Travel&page=1&media=video&yearStart=1920&yearEnd=2019`
It also returned 403. The thumbnail appears for a second and then disappears. 
**Error request:**
`https://images-assets.nasa.gov/video/The%20N.E.X.T.%20Thing%20for%20Space%20Travel/collection.json`
My guess is it has to do with the dots in the id.

**Comment by JustinGOSSES** [comment]
Thanks @prabint . I've sent this to the appropriate party internally.

**Comment by charlescbeebe** [comment]
@prabint dev team's starting to look into it, now! Stay tuned.

**Comment by shentonfreude** [comment]
He @prabint -- I'm able to reproduce the problem you're seeing and am digging into the code to see what's going  on.  More when I find the cause. Thanks for the detailed problem report.

**Comment by shentonfreude** [comment]
OK @prabint 
We've just confirmed in our dev environment that a test file with the exact name you submitted -- including dots and spaces -- publishes and is searchable, so this is not the issue. In this case, the 403 from S3 actually represents a 404 Not Found. We've notified the file's owner and will advise you whether they choose to remove the partial hit from the search results or to re-upload it so it'll be retrievable from S3 again.
